README.md

```diff
@@ -34,6 +34,18 @@
 
 ***
 
+<p align="justify">
+  Currently Thierry Volpiatto is (mostly) the only one still maintaining Helm (<a href="https://sachachua.com/blog/2018/09/interview-with-thierry-volpiatto/">a interview</a> with him)
+  This is a lot of work which Thierry have done freely since 2011. It takes a lot of his time and starts to be more and more difficult
+  maintaining it without financial help. Due to that, the development of Helm is stalled and in maintenance mode by now. 
```

Comment:
Hello René, We can't say that the development is indeed stalled as it was 1 year ago, I am still maintaining helm and did a significant work in the last months, so perhaps you could remove or rephrase this sentence?
Thanks for your work on this and your constant helm support.